packages/mobile_app/lib/common/ambiance/ambiance_subject_settings.dart

```diff
@@ -0,0 +1,175 @@
+      ),
+      child: ListView(
+        padding: const EdgeInsets.all(0.0),
+        shrinkWrap: true,
+        // we need scrolling if user will be in the album orientation or screen is too small
+        scrollDirection: Axis.vertical,
+        children: [
+          // title
+          Container(
+            height: 45.0,
+            decoration: BoxDecoration(
+              borderRadius: BorderRadius.vertical(top: Radius.circular(8.0)),
+              color: AppColors.primaryDark,
+            ),
+            child: Center(
+              child: Text(
+                name.wrapped,
+                style: AppTextStyle.popupTitle,
+              ),
+            ),
+          ),
+          // delete
+          Padding(
+            padding: const EdgeInsets.only(
+              top: 16.0,
+              bottom: 8.0,
+            ),
+            child: GradientBorderButton(
+              child: Padding(
+                padding: const EdgeInsets.symmetric(horizontal: 16.0),
+                child: Text(
+                  localeText.delete.toUpperCase(),
+                  style: AppTextStyle.normalText,
+                ),
+              ),
+              gradient: LinearGradient(
+                colors: [
+                  AppColors.accentDark,
+                  AppColors.accent,
+                  AppColors.primary,
+                ],
+                begin: Alignment.topCenter,
+                end: Alignment.bottomCenter,
+              ),
+              borderAsPadding: const EdgeInsets.all(1.0),
+              background: AppColors.primaryDark,
+              internalPadding:
+                  const EdgeInsets.symmetric(horizontal: 38, vertical: 12.0),
+              borderRadius: BorderRadius.all(Radius.circular(32.0)),
+              onPressed: () {
+                //
+
+                AppGlobal.data.usersRepo.current
+                    .removeAmbianceSubject(widget.subject);
+                AppGlobal.data.usersRepo.write();
+
+                widget.onComplete();
+              },
+            ),
+          ),
+          // user input
+          Padding(
+            padding: const EdgeInsets.symmetric(horizontal: 22.0),
+            child: AmbianceUserInputFields(
+              name: name,
+              role: role,
+              day: day,
+              month: month,
+              year: year,
+              sex: sex,
+            ),
+          ),
+          const SizedBox(
+            height: 18.0,
+          ),
+          GradientFlatButton(
+            onPressed: () {
+              if (name.wrapped.isEmpty ||
+                  role.wrapped.isEmpty ||
+                  day.wrapped.isEmpty ||
+                  month.wrapped.isEmpty ||
+                  year.wrapped.isEmpty ||
+                  int.parse(month.wrapped) > 12 ||
+                  int.parse(month.wrapped) < 1 ||
+                  int.parse(day.wrapped) > 31 ||
+                  int.parse(day.wrapped) < 1) return;
```

Comment:
See my previous comment about validation. Also, it can be shared between `AmbianceSubjectChange` and `AmbianceSubjectNew`